Samples.md

```diff
@@ -0,0 +1,39 @@
+A `sample` in ID3C is "[A sample collected from an individual during a specific encounter](https://github.com/seattleflu/id3c/blob/c5ee5b8d9dbd87a89213f5044a1632cecefd4e7f/schema/deploy/warehouse/sample.sql#L23)."
+See the [warehouse schema diagram] to visualize how samples are related to encounters and other concepts in ID3C.
+
+## REDCap projects
+We wait to create Samples in ID3C until the following conditions are met for each project:
+
+### [SCAN]
+- Mail
+  - `post_collection_data_entry_qc` is complete and verified
+  - `return_utm_barcode` is not blank, or `utm_tube_barcode_2` is not blank, or `reenter_barcode` == `reenter_barcode_2`
```

Comment:
We do not check `utm_tube_barcode_2` or `reenter_barcode` if `return_utm_barcode` field exists in project.

Reply:
Right, therefore it's an `or` statement where the first condition that is met allows the sample to pass

Reply:
I mean if the `return_utm_barcode` field exists at all (blank or not), we do not look at the other fields.

Reply:
Ah yes I see that now. Thanks